app/views/user_mailer/change_email.text.erb

```diff
@@ -1,3 +1,3 @@
 <% content_for :message do %>
-<%= @user.login %>, the email associated with your account has been changed to <%= @user.email %>.
+<%= t('user_mailer.change_email.part1', login: @user.login, email: @user.email) %>
```

Comment:
What's our rationale for not just making it ".part1" and letting Rails put in the view name automatically, to make our code less brittle?